client/comms/server/client.js

```diff
@@ -16,7 +16,18 @@ var Client = function (id_, conn_) {
   this.removed = false;
   this.isTeleporting = false;
   this.loadoutIDX = 0;
-  this.loadout = new Loadout(Loadout.SCOUT);
+  //TODO: Remove the advanced loadouts when we have a way to earn the in-game
+  this.unlockedLoadouts = [
```

Comment:
ok how about this: the inventory of available loadouts is private. client has methods for adding, removing, and maybe resetting (?) loadouts, as well as a method that picks the next loadout (so the server doesn't have to think about the implementation of the loadout list